core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -864,6 +705,5 @@ protected Class defineClassFromData(File container, byte[] classData, String cla
     }
 
     public static boolean useAntClassLoader = SystemProperties.getBoolean(ClassicPluginStrategy.class.getName()+".useAntClassLoader");
-    private static final Logger LOGGER = Logger.getLogger(ClassicPluginStrategy.class.getName());
```

Comment:
CC @varyvol @fcojfernandez  This change likely invalidates the logging tweak in https://github.com/jenkinsci/jenkinsfile-runner/pull/49 / JENKINS-54425 . I am not sure which logger is used in line 692 after this removal